### ISSUE TITLE
api-docs for admin is not clear

### ISSUE THREAD

**Issue by shlomi-nx** [nutanix/papiea-js#311]
The titles do not contain the "provider" prefix, making it a bit hard to follow:

![Screen Shot 2019-10-10 at 1 57 32 PM](https://user-images.githubusercontent.com/39283673/66606809-ba397f80-eb67-11e9-9ff7-822938096cd5.png)

Also, the drop down is great, however, could `/api-docs` direct to a welcome page with a welcome paragraph, followed by a tree of possible pages:

```
Welcome to Papiea's API swagger page. Please select the desired module in the following table:

|- Welcome page
|- Admin apis
|- xi-clusters
    |-1.0.0
    |-1.0.1
```
etc..

The drop down list on top is great to have inside all the pages other than the welcome page.

**Comment by shlomi-nx** [comment]
Related to https://github.com/nutanix/papiea-js/issues/204

**Comment by Emulebest** [comment]
@joshua-berry-ntnx Not sure this is relevant
